mindjunkies/dashboard/migrations/0001_initial.py

```
@@ -15,34 +13,15 @@ class Migration(migrations.Migration):
-                ),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('subject', models.CharField(max_length=255)),
+                ('portfolio_link', models.URLField(blank=True, null=True)),
+                ('certificates', models.FileField(blank=True, null=True, upload_to='teacher_certificates/')),
+                ('is_verified', models.BooleanField(default=False)),
+                ('create_at', models.DateTimeField(auto_now_add=True)),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
             ],
         ),
-    ]
+    ]
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)